gadgets/fsnotify/gadget.yaml

```
@@ -84,32 +84,76 @@ datasources:
         annotations:
           description: Tracee process name
           template: comm
+      tracee_gid:
+        annotations:
+          columns.hidden: "true"
+          description: Tracee group ID
+          template: uid
+          uidgidresolver.target: group
```

To avoid overlaps:
```suggestion
          uidgidresolver.target: tracee_group
```

Ditto in `[tracer|tracee]_[uid|gid]`.